.github/stale.yml

```diff
@@ -7,6 +7,7 @@ daysUntilClose: 7
 exemptLabels:
   - pinned
   - Security
+  - stablebot-ignore
```

Comment:
sorry @robertdown, did you mean this to be `stalebot-ignore`

Reply:
I already commit the fix, did it right to master instead of breaking it to a PR, see [this commit](https://github.com/openemr/openemr/commit/e17a6b74a8a4282cc91a69d828e7ac49e4ce3c77)

Reply:
should probably avoid committing directly to master :sweat_smile:

Reply:
I know... ironically, the typo was in the PR proper, the fix was right to master.